man/man8/zfs.8

```diff
@@ -268,6 +268,21 @@ zfs \- configures ZFS file systems
 .LP
 .nf
 \fBzfs\fR \fBdiff\fR [\fB-FHt\fR] \fIsnapshot\fR \fIsnapshot|filesystem\fR
+.fi
+
+.LP
+.nf
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
```

Comment:
If you do create lots of subcommands, I don't think that makes it necessary to create lots of ioctls; you can keep using one ioctl if that's cleaner/simpler.

> zfs linkkey

Too bad key change doesn't really fit in with the property scheme, otherwise `zfs inherit` would be the obvious choice.  I'd suggest that if we want this functionality, it may make sense to borrow the "inherit" terminology here, e.g. `zfs inheritkey` or `zfs key --inherit`.